package.json

```diff
@@ -40,7 +40,7 @@
     "react-native-device-info": "^0.21.5",
     "react-native-dropdownalert": "^3.4.0",
     "react-native-elements": "^0.19.1",
-    "react-native-firebase": "^5.0.0-rc1",
+    "react-native-firebase": "^4.3.8",
```

Comment:
Is this safe? I thought we needed 5.0.0-rc1 for RN 0.56

Reply:
Yeah.. ideally we'd be using 5.0 :/ Maintainer says 4.3.8 is mostly okay though, a few edge cases where it's rough. https://github.com/invertase/react-native-firebase/issues/1481#issuecomment-421889666

Reply:
The maintainer doesn't specify what the edge cases are though. Are you comfortable with downgrading? Or should we just wait 2 days?

Reply:
Since this is just the foregrounded case I'm okay waiting.

Reply:
down to wait two days i guess. we'll see how accurate the maintainer's time estimate is :P 

if the permissions request stuff turns out to be important for getting notifications on iOS to work, we can just pull them out and land earlier.